ohapi/tests/test_message.py

```diff
@@ -0,0 +1,64 @@
+import os
+import vcr
+import unittest
+import ohapi.api as api
+
+my_vcr = vcr.VCR(path_transformer=vcr.VCR.ensure_suffix('.yaml'),
+                 cassette_library_dir='ohapi/cassettes',
+                 filter_query_parameters=['access_token'])
+
+class test_message(unittest.TestCase):
```

Comment:
As with #35 I'd like to have this moved to `test_api.py`, and renamed to match PEP8 naming guidelines for classes, e.g. APITestMessage